src/mumble/ConnectDialog.h

```diff
@@ -163,7 +163,7 @@ class ServerItem : public QTreeWidgetItem, public PingStats {
 
 		FavoriteServer toFavoriteServer() const;
 		QMimeData *toMimeData() const;
-		static QMimeData *toMimeData(const QString &name, const QString &host, unsigned short port, const QString &channel = QString());
+		static QMimeData *toMimeData(const QString &name, const QString &host, unsigned short port, const QString &channel = QString::fromUtf8("/"));
```

Comment:
TIL. Thought `QLatin1String` was just an alias for `QString::fromLatin1`. I’ll change it.